akka-http-core/src/main/scala/akka/http/impl/engine/client/OutgoingConnectionBlueprint.scala

```diff
@@ -205,14 +204,8 @@ private[http] object OutgoingConnectionBlueprint {
       }
 
       override def onDownstreamFinish(): Unit = {
-        // if downstream cancels while streaming entity,
```

Comment:
Do we actually need to change that? The previous behavior meant that cancelling the *response* stream would keep open the connection until the entity stream was done with. That seems sensible.